docs/docs/ecommerce-tutorial/index.md

```diff
@@ -0,0 +1,542 @@
+     // key: 'pk_test_YOUR_KEY_GOES_HERE',
+     key: 'pk_test_kuhbxb0MMZsp6fj6aTNDnxUu',
+     closed: () => {
+       this.resetButton()
+     },
+   })
+ }
+
+```
+After you replace `pk_test_YOUR_KEY_HERE` with your actual pk_test key and save the change, your button should launch the Stripe Checkout modal and allow you to enter in payment information. You can try and submit a payment, but it will fail because you haven’t set up your serverless function yet. Your Gatsby site is almost ready!
+
+# Setting up a Serverless Function in AWS Lambda
+Lambda is a service offered through Amazon Web Services that allows you to run code that would normally run on a server without having to provision or manage servers. For a simple function like a Stripe charge, Lambda will suit your use case nicely. AWS offers many options for configurations; you are going to use the Serverless Framework to help you minimize the steps to deployment, and also allow you to deploy to other cloud providers if needed.
+
+> **NOTE**: You can follow an adaptation of these steps using this tutorial and Serverless’ docs as a reference if you want to deploy your serverless function to a different provider like Google Cloud or Azure.
+
+## Set up an account through AWS
+
+Before you can set up a serverless function in AWS Lambda, you need an account through AWS. If you don’t already have an account there that you’d like to use, follow these instructions to create an account through AWS. The free tier allows 1 million requests per month. 
+
+## Set up a separate repo
+
+You’ll be setting up a separate repo for the code that you’ll deploy to Lambda. To look through the example repo, inspect the code in GitHub.
+
+ 
+
+Clone the code to a new location on your computer and then change directories into this new folder: 
+
+```shell
+`git clone https://github.com/gillkyle/gatsby-stripe-serverless-backend.git`
+`cd gatsby-stripe-serverless-backend`
+```
+
+Then you need to run `npm install` to install the dependencies in the `package.json` file, which in this case is just the Stripe library.
+
+```shell
+npm install
+```
+
+## What did you just do?
+
+By running `npm install`, you’ve created a node_modules folder that you’ll upload to AWS along with your code to make a charge. All of the code in this file will be hosted online by Amazon, and you need to provide it with the libraries you utilize in your project. By making this repository separate from our Gatsby project, we can decouple it from our site making it easier to switch to a different cloud hosting provider, and greatly decreasing the size of the files we upload to Amazon’s servers.
+
+## Edit your new repo
+
+Open gatsby-stripe-serverless-backend in your code editor.
+
+Rename the `secrets.example.json` file to `secrets.json`.
+Replace the string that says "sk_test_STRIPE_SECRET_KEY" in `secrets.json` with your secret key from your Stripe account, and keep the quotation marks around it
```

Comment:
does it matter if this is test vs production version?

Reply:
The test version will allow orders to go through without needing a real credit card.